mppsolar/__init__.py

```diff
@@ -175,8 +175,8 @@ def main():
 
     args = parser.parse_args()
     prog_name = parser.prog
-    if prog_name is None:
-        prog_name = "mpp-solar"
+#    if prog_name is None:
```

Comment:
this bit would break the jkbms command (or at least change it)
Is there a reason to remove it?

Reply:
Dear @jblance 
To quote you "I think there was a reason..". This was a not removed test.
Cheers,
Volker

Reply:
This bit differentiates between and mppsolar and jkbms call. It is commented out in this PR. Did you mean to?

Reply:
No, please roll back.